src/chain_spec.rs

```diff
@@ -212,24 +241,56 @@ fn testnet_genesis(
             balances: endowed_accounts
                 .iter()
                 .cloned()
-                // endowed in nano, for 1,000,000,000 Dev (=1,000,000,000 Rad)
-                .map(|k| (k, 10_000_000_000_000_000_000))
+                .map(|k| (k, ENDOWMENT))
+                .chain(initial_authorities.iter().map(|x| (x.0.clone(), STASH)))
                 .collect(),
             vesting: vec![],
         }),
+        // TODO do we want/need indices? See substrate's `node/cli/src/chain_spec.rs`
+        session: Some(SessionConfig {
+			keys: initial_authorities.iter().map(|x| {
+				(x.0.clone(), session_keys(x.2.clone(), x.3.clone(), x.4.clone()))
+			}).collect::<Vec<_>>(),
+		}),
+		staking: Some(StakingConfig {
+            // The current era index.
+			current_era: 0,
+            // Minimum number of staking participants before emergency conditions are imposed.
+			minimum_validator_count: 1,
```

Comment:
Is the minimum number we have decided upon ?

Reply:
No, but this parameter has been added to the parameter discussion spreadsheet: https://docs.google.com/spreadsheets/d/1eGc-jEMWLs5qzgYRDPo7VnZgJgWwRn_GjK4v1scuVCg/edit#gid=0